lib/tokenizer.ts

```diff
@@ -17,10 +17,12 @@ export const tokenizer = (regexpStr: string): Root => {
   let last: Token[] = start.stack;
   let groupStack: (Group | Root)[] = [];
 
+  let referenceQueue: Reference[] = [];
+  let groupCount = 0;
+
   const repeatErr = (col: number) => {
     throw new SyntaxError(
-      `Invalid regular expression: /${
-        regexpStr
+      `Invalid regular expression: /${regexpStr
```

Comment:
this PR has many small stylistic changes like this. were these auto formatted? i ran the `lint:fix` script but it didn't generate these changes. even so, let's try to keep this PR's changes related to only the PR's purpose. otherwise, it is harder to read

Reply:
This was probably the prettier extension I have in vscode. I have now reverted these formatting changes :)